server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -670,6 +670,19 @@
             <cryptocurrency>ECA</cryptocurrency>
         </ratesource>
         <cryptologo cryptocurrency="ECA" file="eca.png"/>
+    </extension>
+            <extension class="com.generalbytes.batm.server.extensions.extra.lisk.LiskExtension" >
+        <wallet prefix="liskBinance" name="Lisk">
+            <param name="address" />
+            <param name="binanceApiKey" />
+            <param name="binanceApiSecret" />
+            <cryptocurrency>LSK</cryptocurrency>
+        </wallet>
+        <ratesource prefix="binanceFix" name ="Fix Price" >
```

Comment:
you probably meant `lskFix` or `liskFix`. Also, please make sure it's a unique prefix (goes for the other prefixes as well) in the XML file (should be ok as there is no other LSK implementation).

Reply:
Thank you for your comment, I will fix it within one hour.
Thank you again!